src/command.rs

```diff
@@ -173,6 +191,7 @@ impl fmt::Display for Command {
             Command::ReloadConfig => "reload".to_string(),
             Command::Insert(_) => "insert".to_string(),
             Command::NewPlaylist(name) => format!("new playlist {}", name),
+            Command::Sort(key, direction) => format!("sort playlist by {} {}", key, direction),
```

Comment:
These should represent the command in a way it can be bound to a key or entered within the console, so I'd assume it's a little more like this:

```suggestion
            Command::Sort(key, direction) => format!("sort {} {}", key, direction),
```